### DISCUSSION TITLE
user/update-data

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#23] [Announcements]
# user/update-data

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

http://localhost:8189/user/update-data?idU=gAATe0YRy9LE8FLTnA_Db&update-type=ji-tang

**Comment by wurencaideli** [comment]
可以在此处投稿哟
